syntax = "proto3";

service Showtime {
    rpc GetScheduleByDate(ScheduleDate) returns (ScheduleData) {}
    rpc GetSchedule(Empty) returns (stream ScheduleData) {}
}

message ScheduleDate {
    string date = 1;
}

message ScheduleData {
    string date = 1;
    repeated string movies = 2;
}

message Empty {
}